setup.cfg

```
@@ -1,6 +1,6 @@
 [metadata]
 name = eclcli
-version = 3.5.0
+version = 3.6.0
```

```suggestion
version = 3.7.0
```
赤石澤さんが [IF-9691](https://nttcom.atlassian.net/browse/IF-9691) で `3.5.0 -> 3.6.0` へのバージョンアップをしています。
したがって、こちらでは `3.7.0` にバージョンアップする必要があります。

---

`3.7.0` に修正いたしました。